src/bt_peer_connection.cpp

```diff
@@ -358,6 +358,51 @@ namespace libtorrent {
 		stats_counters().inc_stats_counter(counters::num_outgoing_dht_port);
 	}
 
+	void bt_peer_connection::send_piece_message(message_type const type,
+			counters::stats_counter_t const counter,
```

Comment:
please just use one additional level of indentation, and when breaking a line, please put the "," at the beginning of the new line, for consistency